docs/SolaX-X1.md

```diff
@@ -0,0 +1,80 @@
+<a id="top">
+# SolaX Power - Single phase string inverter X1
+## Compiling
+!!! failure "This feature is not included in precompiled binaries"
+
+To use it you must [compile your build](Compile-your-build). Add the following to `user_config_override.h`:
+```
+#ifndef USE_SOLAX_X1
+#define USE_SOLAX_X1
+```
+
+There are additional `#define` compiler directive parameters which can be used, if they are necessary for your setup.  
+Set an other serial speed than the default value of 9600:
+```
+#undef  SOLAXX1_SPEED
+#define SOLAXX1_SPEED  9600
+```
+If your inverter has two PV-inputs, you can activate the 2nd one:
+```
+#ifndef SOLAXX1_PV2
+#define SOLAXX1_PV2
+```
+## General
+This module reads runtime values from a [Solax X1](https://www.solaxpower.com) device via RS485 Modbus interface and publishes them to MQTT.  
+<img src="/docs/_media/solax-x1/X1Mini.png" width=200> <img src="/docs/_media/solax-x1/X1Air.png" width=200> <img src="/docs/_media/solax-x1/X1Smart.png" width=200>
```

Comment:
@barbudor I've changed the paths and the syntax. A disadvantage of the md-syntax is, that you cannot set the size and align of an image. So I resized the images themself. At one point I left the html-syntax, because it's the only way to show an image beside a table.

Reply:
Correct @SteWers 
Thanks for the doc 👍